rust/main/agents/relayer/src/relayer.rs

```diff
@@ -1255,7 +1256,7 @@ impl Relayer {
         chain_metrics: &ChainMetrics,
     ) -> HashMap<HyperlaneDomain, Arc<dyn ApplicationOperationVerifier>> {
         settings
-            .build_application_operation_verifiers(settings.origin_chains.iter(), core_metrics)
+            .build_application_operation_verifiers(settings.destination_chains.iter(), core_metrics)
```

Comment:
given that in practice `destination_chains` is identical to `origin_chains`, will this change any behavior?